support-models/build.sbt

```diff
@@ -17,3 +17,5 @@ libraryDependencies ++= Seq(
   "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
   "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion
 )
+
+resolvers ++= Seq(Resolver.bintrayRepo("guardian", "ophan")) //event producer library
```

Comment:
this was previously added to all projects, but actually only support-models seems to need it (not even support-workers)